_posts/2023-07-07-newpipe-net-dmca-google-search.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "newpipe.net removed from Google search results due to DMCA take down request"
+short: "newpipe.net removed from Google search results"
+date: 2023-07-07 15:00:00 + 02:00
+categories: [pinned, announcement]
+author: <a href="https://github.com/theScrabi">@theScrabi</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+Dear NewPipe community and other beloved creatures,
+
+Maybe you have already noticed this: if you search for "NewPipe" via Google, you will find plenty information about the project, but a link to the official website is missing. This is because Google submitted to a DMCA takedown notice from a French record label, "Because Music". The notice demands removing our homepage [newpipe.net](https://newpipe.net) from their search listings along with a set of other domains completely unrelated to our project. The DMCA takedown notice was published [in the Lumen database](https://lumendatabase.org/notices/34149383) by Google themselves.
```

Comment:
```suggestion
You might have already noticed this: Searching "NewPipe" via Google will yield plenty information about the project, but a link to the official website is missing. This is because Google submitted to a DMCA takedown notice from a French record label, "Because Music". The notice demands removing our homepage [newpipe.net](https://newpipe.net) from their search listings along with a set of other domains completely unrelated to our project. The DMCA takedown notice was published [in the Lumen database](https://lumendatabase.org/notices/34149383) by Google themselves.
```